base/install_modules.sh

```diff
@@ -2,43 +2,7 @@
 
 set -ex
 
-download_github_module() {
-    github_org=$1
-    module_name=$2
-    tag=$3
-
-    lnls-get-n-unpack -l https://github.com/$github_org/$module_name/archive/refs/tags/$tag.tar.gz
-
-    mv $module_name-$tag $module_name
-}
-
-install_module() {
-    module_name=$1
-    dependency_name=$2
-    release_content="$3"
-
-    cd $module_name
-    echo "$release_content" > configure/RELEASE
-
-    make -j${JOBS} install
-    make clean
-
-    cd -
-
-    echo ${dependency_name}=${EPICS_MODULES_PATH}/${module_name} >> ${EPICS_MODULES_PATH}/../RELEASE
-}
-
-# Install module from GitHub tagged versions or URL
-install_github_module() {
-    github_org=$1
-    module_name=$2
-    dependency_name=$3
-    tag=$4
-    release_content="$5"
-
-    download_github_module $github_org $module_name $tag
-    install_module $module_name $dependency_name "$release_content"
-}
+. ./install-functions.sh
```

Comment:
If epics-base is using an absolute path, why isn't this one?

Edit: I see you fix that in the following commit; please squash it into this first one.

Reply:
Oops. Let this pass by mistake. Good catch.